crates/mdman/src/hbs.rs

```diff
@@ -24,14 +24,13 @@ pub fn expand(file: &Path, formatter: FormatterRef) -> Result<String, Error> {
     handlebars.register_template_file("template", file)?;
     let includes = file.parent().unwrap().join("includes");
     handlebars.register_templates_directory(".md", includes)?;
-    let mut data: HashMap<String, String> = HashMap::new();
     let man_name = file
         .file_stem()
         .expect("expected filename")
         .to_str()
         .expect("utf8 filename")
         .to_string();
-    data.insert("man_name".to_string(), man_name);
+    let data: HashMap<String, String> = [("man_name", man_name)].into();
```

Comment:
Could this perhaps be `HashMap::from([...])` to avoid the need to have type ascription?

Reply:
I will switch (them all) to that in the morning. It is much clearer.